ci/conda/recipes/morpheus/morpheus_build.sh

```diff
@@ -36,7 +37,13 @@ CMAKE_ARGS="-DCMAKE_INSTALL_PREFIX=$PREFIX ${CMAKE_ARGS}"
 CMAKE_ARGS="-DCMAKE_INSTALL_LIBDIR=lib ${CMAKE_ARGS}"
 CMAKE_ARGS="-DBUILD_SHARED_LIBS=ON ${CMAKE_ARGS}"
 CMAKE_ARGS="-DMORPHEUS_USE_CONDA=ON ${CMAKE_ARGS}"
-CMAKE_ARGS="-DMORPHEUS_USE_CCACHE=ON ${CMAKE_ARGS}"
+
+if [[ "${USE_SCCACHE}" == "" ]]; then
+   CMAKE_ARGS="-DMORPHEUS_USE_CCACHE=ON ${CMAKE_ARGS}"
+else
+   CMAKE_ARGS="-DMORPHEUS_USE_CCACHE=OFF ${CMAKE_ARGS}"
+fi
+
```

Comment:
I think there might be a better way to configure using sccache. The script `cmake/setup_cache.cmake` does some templating to ensure ccache can figure out the compiler type during a conda-build run (look at the comments in that file for more info).

We could also just set the variable `CCACHE_PROGRAM_PATH` to `sccache` if they are drop in replacements for eachother.

Either way, we should make a new issue to explore this.

Reply:
Opened issue #110. Resolve this once read